ecommerce-demo/dashboardPlugin/meta.json

```diff
@@ -0,0 +1,5 @@
+{
+    "pluginUrl": "https://gdc-dev-dashboard-plugins.s3.amazonaws.com/internal/dev/dedo/foceasting-plugin/dp_gd_deepar_plugin.js",
+    "pluginName": "gd_deepar",
+    "dashboardIds": ["092929af-375a-4e9c-964f-2add8cdbd259"]
```

Comment:
I think we cannot hard-code dashboardId in the public demo because the id will change each time we create a new demo workspace, won't it?

Reply:
Also, I think we should also define a field that tells us which insight prefix the plugin will work with, shouldn't we?

Reply:
- `dashboardId` is a hardcode in `workspaceAnalytics.json` file, so it is always a fixed value.
-  We don't need a suffix or prefix for Plugin, because Plugin is usually based on the title of Insight or Widget.
(Ex: https://github.com/gooddata/gooddata-public-demos/blob/d9c7ba94e3f2a7aa64a02741e8c92146fff2e4bf/ecommerce-demo/workspaces/demo/workspaceAnalytics.json#L3382
https://github.com/gooddata/gooddata-public-demos/blob/d9c7ba94e3f2a7aa64a02741e8c92146fff2e4bf/ecommerce-demo/workspaces/demo/workspaceAnalytics.json#L421)